schema/rdl_schema_0.1.json

```
@@ -1025,6 +1025,37 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Temporal": {
+			"title": "Temporal coverage",
+			"type": "object",
+			"description": "The temporal period that the dataset covers.",
+			"properties": {
+				"start": {
+					"title": "Start date",
+					"type": "string",
+					"description": "The date on which the period begins. This can refer to a year (YYYY), a month (YYYY-MM) or a day (YYYY-MM-DD) as appropriate for the dataset.",
+					"format": "date-time"
```

`date-time` format is for full ISO8601 date times (YYYY-MM-DDTHH:MM:SSZ). `date` format is for YYYY-MM-DD, but we need to use regex for YYYY-MM and YYYY. My regex skills are not all that so it would be good to get a developer to check this! Apols for the poor indentation.

```suggestion
					"oneOf": [
					          {
					            "format": "date"
					          },
					          {
					            "pattern": "\\d{4}$"
					          },
					          {
					            "pattern": "\\d{4}-(0[1-9]|1[0-2])$"
					          }
					        ]
```

---

ran these past Michael and he gave them the ok :)